### PR TITLE
Feature/publish pkg

### PR DESCRIPTION
**Pull request by kwarunek**
1. Reformated README (to rst, pypi likes it more)
2. Flake on collector
3. travis

### CODE REVIEW

**Review by srespondek — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def read(name):
 
 setup(
     name='diamond-ifcbondcollector',
-    version='2.0.0',
+    version='2.0.1',
```
Comment (evemorgen) [inline setup.py:14]: What about some sort of changelog?

File: MANIFEST.in [hunk 2]
```diff
@@ -1,5 +1,13 @@
-include *.txt
-include *.md
-include conf/IFCBondCollector.conf
```
Comment (evemorgen) [inline MANIFEST.in:3]: are these `conf` files no longer needed?

File: src/ifcbondcollector.py [hunk 3]
```diff
@@ -101,69 +103,79 @@ def get_link_stat(self, ifc_list):
...
         for ifc_name in ifc_list.keys():
-            dev_list.append(ifc_list.get(ifc_name,{}).get('lldp_stats',{}).get('chassis_name',{}))
+            dev_list.append(ifc_list.get(ifc_name, {}).get('lldp_stats', {}).get('chassis_name', {}))
 
-        self.publish_data['mismatch_bond'] = ( dev_list[1:] == dev_list[:-1] )
+        self.publish_data['mismatch_bond'] = dev_list[1:] == dev_list[:-1]
 
+    def _publish(self, name, value):
+        self.publish(name, value)
+        logging.debug('%s %s', name, value)
```
Comment (evemorgen) [inline src/ifcbondcollector.py]: It would be nice if it was more descriptive (also the question is whether it was left here on purpose)

Comment (kwarunek) [reply src/ifcbondcollector.py]: You're right. Done

File: MANIFEST.in (hunk 2 above)
Comment (kwarunek) [reply MANIFEST.in:3]: done

File: setup.py (hunk 1 above)
Comment (kwarunek) [reply setup.py:14]: done

**Review by evemorgen — APPROVED** [review]

**Review by tarfik — APPROVED** [review]

**Pull request merged**